clang/include/clang/StaticAnalyzer/Checkers/SValExplainer.h

```diff
@@ -29,6 +32,17 @@ class SValExplainer : public FullSValVisitor<SValExplainer, std::string> {
   ASTContext &ACtx;
   ProgramStateRef State;
 
+  std::string printCFGElementRef(CFGBlock::ConstCFGElementRef ElemRef) {
```

Comment:
Having the `CFGBlock::ConstCFGElementRef` qualified name spelled every time could get messy really quick.
I didn't count how many times this is spelled throughout the PR, but it's gotta be a lot.

I've just checked that forward declaring is not really an option, and hoisting could be also quite a bit of work. However, we could at least have a type alias to avoid the `CFGBlock::` being spelled out every time.

Could you put `using ConstCFGElementRef = CFGBlock::ConstCFGElementRef;` after the definition of `CFGBlock`?